app/assets/javascripts/admin/addon/components/admin-flags-form.gjs

```diff
@@ -164,6 +168,20 @@ export default class AdminFlagsForm extends Component {
             />
           </div>
 
+          <div class="control-group">
+            <label class="checkbox-label admin-flag-form__require-reason">
+              <Input @type="checkbox" @checked={{this.require_message}} />
```

Comment:
We prefer to use native `<input>` now, see /t/132480

Reply:
See https://github.com/discourse/discourse/blob/8b8f55c85af8eaaec3c41b3d853a32c7560edeab/app/assets/javascripts/discourse/app/components/track-selected.gjs#L17-L17